ard/cost/orbit_wrap.py

```diff
@@ -13,15 +13,48 @@
 
 
 def generate_orbit_location_from_graph(
-    graph_pyomo,
+    graph,  # TODO: replace with a terse_links representation
     X_turbines,
     Y_turbines,
     X_substations,
     Y_substations,
 ):
+    """
+    go from a optiwindnet graph to an ORBIT input CSV
+
+    convert a optiwindnet graph representation of a collection system and get to
+    a best-possible approximation of the same collection system for
+    compatibility with ORBIT. ORBIT doesn't allow branching and optiwindnet
```

Comment:
I'm concerned about the branching/not branching. I think that until we get orbit branching, or a model that can handle the branches, we should not allow optiwindnet to branch.

Reply:
That sounds like a reasonable approach to me.